java/java.lsp.server/vscode/package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "Apache NetBeans Language Server Extension for Visual Studio Code",
 	"author": "Apache NetBeans",
 	"license": "Apache 2.0",
-	"version": "12.4.0",
+	"version": "12.5.0",
```

Comment:
Please put hire `0.1.0` otherwise we are going to have changes with every local `ant build-vscode-ext` - don't you see `0.1.0` automatically being written into this location with every build?

Reply:
Done